client/src/util/metrics/userTasks.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import { getFormFields, is, parse } from './helpers';
+
+export async function getUserTaskMetrics(file) {
```

Comment:
`getUserTaskMetrics` sounds like general metrics in the scope of user tasks to me, not only for forms. Therefore  let the `getUserTaskMetrics`  return 

```js
{
  form: { /* all the stuff you are detecting */ },
  // other stuff to come in the future
}
``` 

or simply rename it to `getUserTaskFormMetrics´